platform/viewer/src/googleCloud/LocationsList.js

```diff
@@ -60,7 +60,7 @@ class LocationsList extends Component {
 
     const body = (
       <tbody id="LocationList">
-        {this.props.locations.map(this.renderTableRow)}
+        {this.props.locations.filter(location => (location.name.split('/')[3].toLowerCase().includes(this.props.filter.toLowerCase()) || this.props.filter=="")).map(this.renderTableRow)}
```

Comment:
Can we destructure the props higher up so this one-liner is easier to read?

Reply:
done